src/EnergyPlus/DataSurfaces.hh

```diff
@@ -578,9 +580,9 @@ namespace DataSurfaces {
         bool operator==(const SurfaceCalcHashKey &other) const
         {
             return (Construction == other.Construction && Azimuth == other.Azimuth && Tilt == other.Tilt && Height == other.Height &&
-                    Zone == other.Zone && EnclIndex == other.EnclIndex && ExtZone == other.ExtZone && ExtEnclIndex == other.ExtEnclIndex &&
-                    ExtSolar == other.ExtSolar && ExtWind == other.ExtWind && ViewFactorGround == other.ViewFactorGround &&
-                    ViewFactorSky == other.ViewFactorSky &&
+                    Zone == other.Zone && EnclIndex == other.EnclIndex && ExtZone == other.ExtZone && ExtCond == other.ExtCond &&
+                    ExtEnclIndex == other.ExtEnclIndex && ExtSolar == other.ExtSolar && ExtWind == other.ExtWind &&
+                    ViewFactorGround == other.ViewFactorGround && ViewFactorSky == other.ViewFactorSky &&
```

Comment:
I get why this was written this way, but a 12-line long compound if is hard on the eyes. Just saying.

Reply:
Yeah, it would be nice to clean this into a couple individual statements, if there is a way to avoid a performance penalty anyway.  That's for later though unless something else comes up here.